src/System Application/App/Guided Experience/src/Checklist Banner/ChecklistBanner.Codeunit.al

```diff
@@ -41,4 +41,15 @@ codeunit 3728 "Checklist Banner"
     internal procedure OnOpenChecklistBannerPage(var SkipWelcomeState: Boolean; IsEvaluationCompany: Boolean)
     begin
     end;
+
+    /// <summary>
+    /// Integration Event to provide the ability to filter on the checklist items before it is loaded to the checklist banner.
+    /// Do NOT use complex logic in the subscriber as it will impact the performance of opening company.
```

Comment:
seems like we shouldn't have this event at all... Why create the possibility of things going wrong?

Reply:
Agree, trying a few things to try to get rid of it

Reply:
Removed event after swtiching approach